### PR TITLE
Change xkcd comic to 1319

### PR DESCRIPTION
**Pull request by jakirkham**
Checklist
* [x] Added a ``news`` entry

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @ocefpaf (as we discussed this previously)

**Review by msarahan — APPROVED** [review]
definitely a positive change. The old one was funny, but the last bit was not professional enough.

**Comment by ocefpaf** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -134,9 +134,9 @@ $ rever 0.1.2
...
 --------------------------
 
-#### xkcd 1629: Tools
+#### xkcd 1319: Automation
 
-[![xkcd 1629: Tools](https://imgs.xkcd.com/comics/tools.png)](https://xkcd.com/1629/)
+[![xkcd 1319: Automation](https://imgs.xkcd.com/comics/automation.png)](https://xkcd.com/1319/)
 
 **Titletext**: *I make tools for managing job-hunting sites for people who make*
 *tools for managing job-hunting sites for people who make tools for ...*
```
Comment (jakirkham) [inline README.md:142]: Missed removing this. Handling in PR: https://github.com/conda-forge/conda-smithy/pull/1803

**Pull request merged**